src/types/react.ts

```diff
@@ -24,7 +24,7 @@ export type ReactHooksRenderer = {
     options?: RenderHookOptions<TProps>
   ) => RenderHookResult<TProps, TResult>
   act: Act
-  cleanup: () => void
+  cleanup: () => Promise<void>
```

Comment:
As mentioned int he PR description, this is fixing the type to match the implementation.